### ISSUE TITLE
Windows app menu redesign

### ISSUE THREAD

**Issue by niik** [desktop/desktop#836]
After #775 me and @donokuda took a look at the application menu on Windows and decided that we should move it into the title bar and explode it into top-level menu items.

<img width="416" alt="screen shot 2017-01-20 at 10 56 14" src="https://cloud.githubusercontent.com/assets/634063/22161429/255b9df2-deff-11e6-90a6-2a2db74f3928.png">

<img width="455" alt="screen shot 2017-01-20 at 10 57 59" src="https://cloud.githubusercontent.com/assets/634063/22161481/5b815b42-deff-11e6-9e0f-da1d454f1d40.png">

We're also adding a flat version of the app icon (simply the mark) to the title bar to make it feel even more appropriate on the platform. Clicking on the app icon could (although it's not a priority in any way shape or form) show a traditional window controls menu (min, max, etc).
